### DISCUSSION TITLE
Battery charging limit.

### DISCUSSION THREAD

**Discussion by Coffefe** [elementary/triage#554] [Ideas]
Is it possible to add a slider or something to stop charging the battery at select value(50-60-70-80%) ?
Would be nice because I use the laptops mostly plugged and I am scared of bulging batteries(also might help other users like me).

**Comment by UncleTallest** [comment]
If more information is needed to code this, take a look at the mechanisms that `tlp` uses for this.

Project GitHub => https://github.com/linrunner/TLP

Charge stop/start => https://linrunner.de/tlp/settings/battery.html#start-stop-charge-thresh-batx

Vendor-specific => https://linrunner.de/tlp/settings/bc-vendors.html

**Comment by 1280px** [comment]
There is a way to do this without GUI, but having it as a separate option in the Switchboard plug would be nice.

Someone also opened an issue about it just a few days before your thread: https://github.com/elementary/switchboard-plug-power/issues/270

**Comment by lenemter** [comment]
Closing as duplicate of https://github.com/elementary/switchboard-plug-power/issues/270
